stellarphot/notebooks/photometry/06-transit-fit-template.ipynb

```diff
@@ -63,7 +70,32 @@
     "photometry_file = fo.path\n",
     "\n",
     "# 👉 File with exoplanet info in\n",
-    "tess_info_output_file = fo2.path"
+    "tess_info_output_file = fo2.path\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 👇👇👇 use this to exclude some data (only if needed!) 👇👇👇"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "use_no_data_before = Time(2400000, format='jd', scale='tdb')\n",
+    "\n",
+    "use_no_data_between = [\n",
+    "    [\n",
+    "        Time(2400000, format='jd', scale='tdb'),\n",
+    "        Time(2400000, format='jd', scale='tdb')\n",
+    "    ]\n",
+    "]\n",
+    "\n",
+    "use_no_data_after = Time(2499999, format='jd', scale='tdb')\n"
```

Comment:
Interesting feature.  And it tells me you are introducing a Y2132 bug. :)

Reply:
Anyone still using this in 2132 will get what they deserver 😉